Sources/Graphaello/Extensions/ExprSyntax+erasure.swift

```diff
@@ -4,8 +4,8 @@ import SwiftSyntax
 
 extension ExprSyntax {
 
-    func withoutErasure() -> ExprSyntaxProtocol {
-        return asProtocol(ExprSyntaxProtocol.self)
+    func switchOver() -> SyntaxEnum {
+        return _syntaxNode.as(SyntaxEnum.self)
```

Comment:
`_syntaxNode` is an underscored member and should not be accessed unless there is evidence that it improves performance in performance-critical code. The preferred spelling is `Syntax(self)`